packages/overlay/src/Overlay.ts

```diff
@@ -883,7 +883,6 @@ export class Overlay extends OverlayFeatures {
     }
 
     public override manuallyKeepOpen(): void {
-        super.manuallyKeepOpen();
```

Comment:
Why was this removed? Not needed, or did it cause problems?

Reply:
No needed and even ghost calls have a performance cost so 💥